spec/commands/newgem_spec.rb

```diff
@@ -116,19 +119,20 @@ def gem_skeleton_assertions(gem_name)
     let(:gem_name) { "test_gem" }
     let(:generated_gem) { Bundler::GemHelper.new(bundled_app(gem_name).to_s) }
 
-    context "git config user.name present" do
+    context "git config github.user present" do
       before do
         execute_bundle_gem(gem_name)
       end
 
       it "contribute URL set to git username" do
         expect(bundled_app("test_gem/README.md").read).not_to include("[USERNAME]")
+        expect(bundled_app("test_gem/README.md").read).to include("github.com/bundleuser")
       end
     end
 
-    context "git config user.name is absent" do
+    context "git config gihhub.user is absent" do
       before do
-        `git config --unset user.name`
+        `git config --unset github.user`
```

Comment:
@danielpclark The cool thing about `sys_exec` is that in `spec_helpers/support.rb` it is defined like this:

```ruby
    def sys_exec(cmd)
      Open3.popen3(cmd.to_s) do |stdin, stdout, stderr, wait_thr|
        yield stdin, stdout, wait_thr if block_given?
        stdin.close

        @exitstatus = wait_thr && wait_thr.value.exitstatus
        @out = Thread.new { stdout.read }.value.strip
        @err = Thread.new { stderr.read }.value.strip
      end

      (@all_output ||= String.new) << [
        "$ #{cmd.to_s.strip}",
        out,
        err,
        @exitstatus ? "# $? => #{@exitstatus}" : "",
        "\n",
      ].reject(&:empty?).join("\n")

      @out
    end
    bang :sys_exec

```

As you can see, some of the noteworthy output-management makes easily-checkable, standardized strings out of the I/O.

There are other helper methods for specs that check that standardized output.

So, using `sys_exec` smooths the path for the RSpec usage in the project.